spec/spec_helper.rb

```diff
@@ -11,6 +11,10 @@
 # Must be explicit as activerecord is optional dependency
 require 'active_record/railtie'
 
+unless File.exist?('config/database.yml')
```

Comment:
ok so I think that's the error @wchen-r7 is seeing. Any idea how to do the check correctly? Just hardcode the path?